go.mod

```diff
@@ -81,6 +81,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require github.com/pelletier/go-toml v1.9.5
```

Comment:
Nit: You could move it to the first require block (contains all direct dependencies).